src/main/java/io/confluent/copycat/hdfs/HdfsWriter.java

```diff
@@ -27,17 +25,27 @@
 import java.io.IOException;
 import java.net.URI;
 import java.util.HashMap;
+import java.util.HashSet;
 import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.ReentrantLock;
 
 import io.confluent.copycat.connector.TopicPartition;
 import io.confluent.copycat.errors.CopycatException;
 import io.confluent.copycat.errors.CopycatRuntimeException;
 import io.confluent.copycat.sink.SinkRecord;
 
-
 public class HdfsWriter {
+
   private static final Logger log = LoggerFactory.getLogger(HdfsWriter.class);
   private Map<TopicPartition, RecordWriter<Long, SinkRecord>> writers = null;
+  private Map<TopicPartition, WAL> wals = null;
```

Comment:
I will address this in a following patch which I will have a writer for each topic partition.